ios/Podfile.lock

```diff
@@ -210,7 +211,7 @@ PODS:
     - React
   - react-native-blob-util (0.13.18):
     - React-Core
-  - react-native-blur (0.8.0):
+  - react-native-blur (3.6.1):
```

Comment:
@Parveshdhull is that ok to change the version here? It's changed after `make shell`

Reply:
Hi @yqrashawn, Thank you very much for PR.
Yes, it's ok. We currently use `react-native-blur` version 3.6.1, so it's expected.